apps/api/lib/api_web/controllers/events_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ApiWeb.EventsController do
+  @moduledoc """
+  Web interface for viewing events
+  """
+
+  use ApiWeb, :controller
+
+  alias Core.EventRepo
+
+  action_fallback(ApiWeb.ErrorsController)
+
+  def index(conn, _opts) do
+    {:ok, events} = EventRepo.events()
+    json(conn, %{data: %{events: events}})
+  end
+
+  def show(conn, %{"id" => raw_id}) do
+    with {:ok, id} <- Helpers.Integer.parse(raw_id),
+         {:ok, events} <- EventRepo.get_event(id) do
+      json(conn, %{data: %{events: events}})
+    end
+  end
+
+  def show(conn, %{"operation" => operation}) do
+    with operation <- String.to_existing_atom(operation),
+         {:ok, events} <- EventRepo.get_event(operation) do
```

Comment:
Here returning multiple events makes sense, so instead of `EventRepo.get_event` a name like `EventRepo.events_for_operation` should probably be used instead.